Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
-name = "crate-template"
+name = "vm-allocator"
 version = "0.1.0"
-authors = [TODO]
+authors = ["rust-vmm AWS maintainers <rust-vmm-maintainers@amazon.com>"]
+description = "Allocation of resources needed by VMM"
+keywords = ["Id", "Address space", "IRQ", "MSI", "MSI-x"]
```

Comment:
You can't use space in keywords, it will fail when you'll try to publish it. I also think the keywords are not speaking so much about what the crate does.

I am thinking about a keyword that speaks more about the main action of the crate "allocating resources" than about what is the type of resources that you can allocate. At this point, I can't come up with anything.

Reply:
This comment about spaces in [keywords](https://doc.rust-lang.org/cargo/reference/manifest.html#the-keywords-field) has not been addressed.

Reply:
I have removed keywords to not block this PR in this. We will add it when we find something to put there.